src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,73 @@
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(4, $card-width);
+  gap: 48px;
+  justify-content: center;
+  grid-auto-flow: row;
+
+  &__card {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: $card-height;
+    border: 1px solid #ccc;
+    border-radius: 5px;
+  }
+}
+
+@media (min-width: 768px) and (max-width: 1023px) {
```

Comment:
It's better to apply Mobile First approach, where you start to create your markup from mobile to desktop devices. 
Because nowadays mobile traffic is about 80% of the whole web
So, please rewrite your queries using this approach

And you don't need to make such segments as (min-width: 488px) and (max-width: 767px)
Just use (min-width: 488px) 
(min-width: 768px)